helm/beeai-platform/templates/docling/docling-serve.yaml

```diff
@@ -0,0 +1,58 @@
+# This example deployment configures Docling Serve with a Service and cuda image
+{{ if .Values.features.docling }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: docling-serve
+  labels:
+    app: docling-serve
+    {{- include "beeai-platform.labels" . | nindent 4 }}
+spec:
+  ports:
+  - name: http
+    port: {{ .Values.docling.service.port }}
+    targetPort: http
+  selector:
+    app: docling-serve
+    component: docling-serve-api
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: docling-serve
+  labels:
+    app: docling-serve
+    component: docling-serve-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: docling-serve
+      component: docling-serve-api
+  template:
+    metadata:
+      labels:
+        app: docling-serve
+        component: docling-serve-api
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: api
+          resources:
+            limits:
+              cpu: {{ .Values.docling.resources.limits.cpu }}
+              memory: {{ .Values.docling.resources.limits.memory }}
+            requests:
+              cpu: {{ .Values.docling.resources.requests.cpu }}
+              memory:  {{ .Values.docling.resources.requests.memory }}
+          env:
+            - name: DOCLING_SERVE_ENABLE_UI
+              value: {{ if .Values.docling.env.DOCLING_SERVE_ENABLE_UI }}'true'{{ else }}'false'{{ end }}
```

Comment:
[nitpick] Let's use something like `.Values.docling.ui.enabled` instead

Exposing all env variables would typically done through `.Values.docling.extraEnv` and by a for loop over all variables, not explicitly mentioning each one here